### DISCUSSION TITLE
import mediapipe flet long time loading

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1672] [Q&A]
### Question

i try import mediapipe but why flet loading long time . whats wrong 

root@miop-HP-Convertible-x360-11-ab1XX:/home/miop# flet --version
0.7.2

### Code sample

```python
from flet import *

import mediapipe as mp
def main(page:Page):
    page.window_width = 300
    page.add()

flet.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Try adding something to the page. 
Some `Hello World` Text for instance.
